scripts/feature_tests/cleanup_env.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -xe
+
+# shellcheck disable=SC1091
+source lib/logging.sh
+# shellcheck disable=SC1091
+source lib/common.sh
+# shellcheck disable=SC1091
+source lib/network.sh
+
+export GOPATH="/home/${USER}/go"
+M3PATH="${M3PATH:-${GOPATH}/src/github.com/metal3-io}"
+BMOPATH="${BMOPATH:-${M3PATH}/baremetal-operator}"
+CAPIPATH="${CAPIPATH:-${M3PATH}/cluster-api}"
+CAPM3_BASE_URL="${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}"
+CAPM3RELEASEPATH="${CAPM3RELEASEPATH:-https://api.github.com/repos/${CAPM3_BASE_URL}/releases/latest}"
+CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}")}"
+NAMESPACE="metal3"
```

Comment:
some of this is duplicated from 03, put it in lib/common.sh instead

Reply:
Done